modules/op-resolver-community-tasks-ref.adoc

```diff
@@ -0,0 +1,411 @@
+|===
+
+.Results that the `jib-maven` task returns
+[cols="1,1,2",options="header"]
+|===
+| Result | Type | Description
+|`IMAGE_DIGEST` |`string` |Digest of the image that was built.
+|===
+
+.Changes from the `jib-maven` community cluster task
+
+* The default values for the `IMAGE` and `MAVEN_IMAGE` parameters were changed.
+
+[discrete]
+[id="op-ctaskref-kubeconfig-creator_{context}"]
+== kubeconfig-creator
+The `kubeconfig-creator` community task creates a `kubeconfig` file that can be used by other tasks in the pipeline to access different clusters.
+
+.Example usage of the `kubeconfig-creator` community task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: kubeconfig-creator
+spec:
+  tasks:
+    - name: kubeconfig-creator
+      params:
+        - name: name
+          value: example_cluster
+        - name: url
+          value: example_cluster_URL
+        - name: username
+          value: example_username
+        - name: password
+          value: example_password
+        - name: cadata
+          value: ''
+        - name: clientKeyData
+          value: ''
+        - name: clientCertificateData
+          value: ''
+        - name: namespace
+          value: ''
+        - name: token
+          value: ''
+        - name: insecure
+          value: 'false'
+      taskRef:
+        kind: Task
+        name: kubeconfig-creator
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+  #...
+----
+
+.Supported parameters for the `kubeconfig-creator` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`name` |The name of the cluster to access. |
+|`url` |The address of the cluster to access. |
+|`username` |The username for basic authentication to the cluster. |
+|`password` |The password for basic authentication to the cluster. |`""`
+|`cadata` |Contains PEM-encoded certificate authority (CA) certificates. |`""`
+|`clientKeyData` |Contains PEM-encoded data from a client key file for TLS. |`""`
+|`clientCertificateData` |Contains PEM-encoded data from a client certification file for TLS. |`""`
+|`namespace` |The default namespace to use on unspecified requests. |`""`
+|`token` |The bearer token for authentication to the cluster. |`""`
+|`insecure` |To indicate whether a server should be accessed without verifying the TLS certificate. |`false`
+|===
+
+.Supported workspaces for the `kubeconfig-creator` community task
+[cols="1,3",options="header"]
+|===
+| Workspace | Description
+| `output` | The workspace where the `kubeconfig` file is stored by the `kubeconfig-creator` task.
+|===
+
+[discrete]
+[id="op-ctaskref-pull-request_{context}"]
+== pull-request
+
+You can use the `pull-request` community task to interact with a source control management (SCM) system through an abstracted interface.
+
+This community task works with both public SCM instances and self-hosted, enterprise GitHub, or GitLab instances. 
```

Comment:
as it probably works *only* with GitHub and GitLab, and not with some other "self-hosted" repo, I'd remove the commas.